config/rbac/role.yaml

```
@@ -6,6 +6,19 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
```

```suggestion
  - get
  resourceNames:
  - spanner-autoscaler-service-account
```

>The downside is that the spanner-autoscaler-controller will have read access to all the secrets.

I have a feeling this will solve the problem. Do you have any concerns?

---

This is a good idea. I have now restricted the default access only the secrets named `spanner-autoscaler-gcp-sa`.

Thanks for the suggestion!